### PR TITLE
feat: Support configure frontmatter classifier scope layout

### PR DESCRIPTION
**Pull request by Coolyang1996**
**Summary**
fix: #23 
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [x] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: docs/guide/getting-started.md [hunk 1]
```diff
@@ -261,13 +263,13 @@ the corresponding layout:
 
 | url         | layout                             |
 | ----------- | ---------------------------------- |
-| `/tag/`     | `Tag` (fallback to `FrontmatterKey` if not exist)  |
-| `/tag/vue/` | `FrontmatterPagination` (fallback to `Layout` if not exist)  |
-| `/tag/js/`  | `FrontmatterPagination` (fallback to `Layout` if not exist)  |
+| `/tag/`     | `Tags` (fallback to `FrontmatterKey` if not exist)  |
+| `/tag/vue/` | `Tag` (fallback to `FrontmatterPagination` or `Layout` if not exist)  |
```
Comment (newsbielt703) [inline docs/guide/getting-started.md]:
```suggestion
| `/tag/vue/` | `Tag` (fallback to `FrontmatterPagination` if not exist)  |
```

File: src/node/handleOptions.ts [hunk 2]
```diff
@@ -133,6 +133,7 @@ export function handleOptions(
       keys,
       path: indexPath,
       layout: indexLayout,
+      scopeLayout = '',
```
Comment (newsbielt703) [inline src/node/handleOptions.ts]:
```suggestion
      scopeLayout,
```

File: src/node/index.ts [hunk 3]
```diff
@@ -129,7 +129,7 @@ module.exports = (options: BlogPluginOptions, ctx: VuePressContext) => {
                 pid: scope,
                 id: key,
                 frontmatter: {
-                  layout: DefaultLayoutEnum.FrontmatterPagination,
+                  layout: scopeLayout || DefaultLayoutEnum.FrontmatterPagination,
```
Comment (newsbielt703) [inline src/node/index.ts]:
```suggestion
                  layout: ctx.getLayout(scopeLayout, DefaultLayoutEnum.FrontmatterPagination),
```

File: src/node/interface/Frontmatter.ts [hunk 4]
```diff
@@ -5,6 +5,7 @@ export interface FrontmatterClassificationPage {
   id: string;
   pagination: PaginationConfig;
   keys: string[];
+  scopeLayout: string;
```
Comment (newsbielt703) [inline src/node/interface/Frontmatter.ts]:
```suggestion
  scopeLayout?: string;
```

**Comment by Coolyang1996** [comment]
> Nice.
> 
> Just some suggestions.

Done.

**Pull request merged**